compass_app/app/lib/main.dart

```diff
@@ -6,12 +6,13 @@ import 'package:provider/provider.dart';
 
 import 'ui/core/ui/scroll_behavior.dart';
 
+/// Configure dependencies in main method.
+/// - Use [providersLocal] to display local data.
+/// - Use [providersRemote] to display remote data from a server.
```

Comment:
Should we set up different app configurations for this?  (Non-blocking comment, this can be implemented later)

Reply:
Good idea, we could setup different main targets e.g. `main_local.dart`, `main_remote.dart` with a default `main.dart`

Reply:
I have implemented a first idea over this, creating `main_development.dart` and `main_staging.dart` in the app lib root.